ci/environment-py37-upstream-master.yml

```diff
@@ -10,3 +9,4 @@ dependencies:
     - docrep
     - codecov
     - pytest-cov
+    - black
```

Comment:
```
  - git+https://github.com/pydata/xarray.git
```
will install the github master into the environment

Reply:
🤦‍♂ that is obviously easier. Will fix that right now.